src/diffusers/models/modeling_utils.py

```diff
@@ -1041,7 +1041,6 @@ def from_pretrained(cls, pretrained_model_name_or_path: Optional[Union[str, os.P
                         model,
                         state_dict,
                         device=param_device,
-                        dtype=torch_dtype,
```

Comment:
Why is this going away?

Reply:
Oh this is a mistake. Thanks for catching.